tensorrt_llm/inputs/utils.py

```
@@ -363,6 +363,15 @@ async def async_load_audio(
     return audio
 
 
+async def async_load_base64_audio(
+    data: str,
+    format: str = "wav",
```

Ruff (0.15.9)

[error] Function argument `format` is shadowing a Python builtin

(A002)